app/tut/ga/cad-e-riomhchlaru.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Cad é Ríomhchlárú?
+prev: reamhra
+prev-text: Réamhrá Teagaisc
+---
+
+# Is é Ríomhchlárú ná...
+
+Sula dtosóimid ag foghlaim conas Setanta a scríobh agus a úsáid, ba chóir dúinn an ceist seo a chur:
+
+> Cad é ríomhchlárú?
+
+Inniu tá ríomhairí an chliste, is féidir leo áirimh a dhéanamh i bhfaiteadh na súl nach féidir le aon duine a dhéanamh ina cheann. Áfach, tá a lán cabhair ag teastáil ó ríomhairí agus is féidir linn iad a chabhrú le ríomhchlárú.
+
+Nuair a scríobhaimid ríomhchláir, scríobhaimid treoracha chun an ríomhaire a leanúint. Léifidh an ríomhaire ár dtreoracha agus leanfaidh sé iad go díreach.
+
+## Cad é teanga ríomhchlárúcháin?
+
+Is é teanga ríomhchlarúcháin ná teanga speisialta a thuigeann ríomhairí. Bainimid úsáid as chun ár dtreoracha a scríobh.
+
+Deartar teangacha ríomhchlárúcháin ionas gur féidir le daoine iad a thuiscint, chomh maith le ríomhairí. Is é *Setanta* ceann de na teangacha sin, ach tá sé beagáinín difriúil.
```

Comment:
is ceann de na teangacha sin é *Setanta*

Reply:
Ceartaithe anois